src/main/java/org/folio/util/ResponseUtils.java

```diff
@@ -24,24 +23,24 @@ public class ResponseUtils {
   private ResponseUtils() {
   }
 
-  public static <T> Handler<AsyncResult<Tx<T>>> handleNoContentResponse(Handler<AsyncResult<Response>> asyncResultHandler, Tx<T> tx,
-    String logMessage) {
-    logger.debug("handleNoContentResponse:: Trying to handle no content response for entity: {}', logMessage: {}", tx.getEntity(), logMessage);
-    return result -> {
-      if (result.failed()) {
-        HttpException cause = (HttpException) result.cause();
-        logger.error(logMessage, cause, tx.getEntity(), "or associated data failed to be");
-
-        // The result of rollback operation is not so important, main failure cause is used to build the response
-        tx.rollbackTransaction().onComplete(res -> asyncResultHandler.handle(buildErrorResponse(cause)));
-      } else {
-        logger.info(logMessage, tx.getEntity(), "and associated data were successfully");
-        asyncResultHandler.handle(buildNoContentResponse());
-      }
-    };
+  public static <T> void handleNoContentResponse(AsyncResult<T> result,
```

Comment:
This method pretends to be som general utility but actually it's not: it contains very specific log messages